src/pages/Committees.js

```diff
@@ -140,17 +190,17 @@ function METAdorerna() {
     return(
         <p>
             METAdorerna (META doors) is the CS-Media committee that takes care of the Chapter Hall, META. If you do
-            not know yet, it is located at Osquars backe 21, 114 28 Stockholm, Sweden (or use this convenient 
+            not know yet, it is located at Osquars backe 21, 114 28 Stockholm, Sweden (or use this convenient{' '}
```

Comment:
`... META, which by now you should know is located at ...` or something; reword so it's clear you're giving META's location and not METAdorerna's specifically